walnut/schemata/Task.json

```diff
@@ -0,0 +1,169 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/obviyus/Controlss/walnut/schemata/Task.json",
```

Comment:
Shouldn't the `$id` be an absolute-URI in a domain which we control? (e.g `https://github.com/AliceO2Group/Control/walnut/schemata/Task.json`)

Reply:
Ultimately I question the need of an ID at all, but if we do define one, what you propose seems correct.